x-pack/plugins/apm/server/routes/metrics/serverless/helper.test.ts

```diff
@@ -38,3 +42,85 @@ describe('calcMemoryUsedRate', () => {
     expect(calcMemoryUsedRate({ memoryFree: 50, memoryTotal: 100 })).toBe(0.5);
   });
 });
+
+const AWS_LAMBDA_PRICE_FACTOR = {
+  x86_64: 0.0000166667,
+  arm: 0.0000133334,
+};
+
+describe('calcEstimatedCost', () => {
+  it('returns 0 when price factor is not defined', () => {
```

Comment:
why does this return 0 rather than a null-ish value?

Reply:
Yeah, it does make sense to return undefined instead of 0